### DISCUSSION TITLE
Created Account But Registration Confirmation Email Never Arrived

### DISCUSSION THREAD

**Discussion by mtp1032** [chromiecraft/chromiecraft#2394] [General]
I created a new account, _terry148_ (along with my email address) but the confirmation email never arrived. I tried to login to the game today in the hopes that the account was validated anyway. Alas, no joy.

What can I do to get this account validated?

Thanks, in advance.
P.S. Yes. I have checked the Trash and Spam folders.

**Comment by 55Honey** [comment]
Hi mtp1032,
There is no account with that username. Please retry registering. You can add an alias to an email to create more accounts with the same mail.
E.g. 
`yourmail+chromie@gmail.com`
`yourmail+ccacount@gmail.com`
`yourmail+cc1@gmail.com`

**Reply by mtp1032** [reply]
Fascinating. Reregistration failed with the message that the username and email address were taken. I'll try registering via an alias as you suggest. But, I suspect it won't work since I never received the confirmation email in the first place.

Thanks

**Reply by mtp1032** [reply]
Thank you. Registering with an alias worked just fine.

Cheers,
